src/geometry.rs

```diff
@@ -45,6 +49,18 @@ impl Geometry {
     }
 }
 
+impl PartialEq for Geometry {
+    fn eq(&self, other: &Self) -> bool {
+        self.to_wkt().unwrap() == other.to_wkt().unwrap()
+    }
+}
+
+impl PartialOrd for Geometry {
+    fn partial_cmp(&self, _other: &Self) -> Option<Ordering> {
+        Some(Ordering::Equal)
+    }
+}
+
```

Comment:
Why do we need this noop impl?

Reply:
It simplified things so that I could just derive PartialEq and PartialOrd on Expr to use those when using EQOPS or CMPOPS.

Reply:
Does it give false results? Will people be surprised when a geometry over California is "equal" to a geometry over Greenland?

Reply:
OK, I've now made PartialEq actually use s_equals. PartialOrd is still there, but bails using the unimplemented! macro

Reply:
/giphy notsure

I'd prefer to not bake in a panic into that common of a code path. Can't you return `None` from `partial_cmp` for this case? https://doc.rust-lang.org/std/cmp/trait.PartialOrd.html

Reply:
I just YOLO'd it 😄